dev-docs/examples/jw-cloud-example.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: player_example
+title: JW Cloud Player Integration
+description: Show a prebid ad in a cloud hosted JW Player
+
+top_nav_section: dev_docs
+nav_section: quick-start
+
+hide: true
+beta: true
+
+about:
+- Setting up JW Player to dynamically play ads.
+- Using a cloud-hosted JW player.
+- Using invokeVideoPlayer to set up and access the JW Player instance.
+- Dynamically inserting a preroll ad into JW Player.
+
+player_notes:
+- You must have the correct JW Player license that allows you to play advertising.
+- The different methods of embedding JW Player on your site can be found <a href="https://support.jwplayer.com/customer/portal/articles/1406723-mp4-video-embed">here</a>. 
+- For this example we will be using method 1, a cloud-hosted player and JW Platform hosted content. To see an example using the self-hosted player, click <a href="#">here</a>.
```

Comment:
This link doesn't point to anything - I assume you meant to point to a hosted example somewhere?